tests/libres_tests/res/enkf/test_enkf_load_results_manually.py

```diff
@@ -70,3 +73,26 @@ def test_load_results_from_run_context(self):
             self.assertEqual(24, loaded)
             self.assertEqual(25, len(expected))
             self.assertEqual(25, len(realisations))
+
+
+class TestLoadResultsManuallyLogtest:
+    """
+    This little test does not depend on Equinor-data and only verifies
+    the lazy_load flag in forward_load_context plus memory-logging
+    """
+
+    def test_load_results_manually(self, setup_case, caplog, monkeypatch):
+        for val in ["false", "true"]:
+            res_config = setup_case("local/snake_oil", "snake_oil.ert")
+            ert = EnKFMain(res_config)
+            load_from = ert.getEnkfFsManager().getFileSystem("default_0")
+            ert.getEnkfFsManager().switchFileSystem(load_from)
+            realisations = BoolVector(default_value=False, initial_size=25)
+            realisations[0] = True  # only need one to test what we want
+            with caplog.at_level(logging.INFO):
+                loaded = ert.loadFromForwardModel(realisations, 0, load_from)
+                assert 0 == loaded  # they will  in fact all fail, but that's ok
+                assert f"lazy={val}" in caplog.text
+
+            # Set env-var for second iteration
+            monkeypatch.setenv("ERT_LAZY_LOAD_SUMMARYDATA", "true")
```

Comment:
You can have it as a class, but dont quite see the benefit here:
```suggestion
@pytest.mark.parametrize("lazy_load", [True, False])
def test_load_results_manually(setup_case, caplog, monkeypatch, lazy_load):
    """
    This little test does not depend on Equinor-data and only verifies
    the lazy_load flag in forward_load_context plus memory-logging
    """
    if lazy_load:
        monkeypatch.setenv("ERT_LAZY_LOAD_SUMMARYDATA", lazy_load)

    res_config = setup_case("local/snake_oil", "snake_oil.ert")
    ert = EnKFMain(res_config)
    load_from = ert.getEnkfFsManager().getFileSystem("default_0")
    ert.getEnkfFsManager().switchFileSystem(load_from)
    realisations = BoolVector(default_value=False, initial_size=25)
    realisations[0] = True  # only need one to test what we want
    with caplog.at_level(logging.INFO):
        loaded = ert.loadFromForwardModel(realisations, 0, load_from)
        assert 0 == loaded  # they will  in fact all fail, but that's ok
        assert f"lazy={lazy_load}".lower() in caplog.text
```

Reply:
Fair enough